tools/run-dev.py

```diff
@@ -37,6 +39,13 @@
 behavior, the reverse proxy itself does *not* automatically restart on changes
 to this file.
 """)
+if __name__ == "__main__":
```

Comment:
I think you can omit this "if" statement for now.  We only run this program as a script, not a module.  We should eventually wrap a lot more of this program in `if __name__ == "__main__"`, but I'm not sure this is the commit where we start this.